package.json

```diff
@@ -9,7 +9,7 @@
     "start": "next start",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "lint": "eslint . --ext=tsx,ts",
+    "lint": "eslint . --ext=tsx,ts,json --fix",
```

Comment:
Thoughts on breaking this out into a `lint:fix` script that will apply fixes? Seems a bit wonky to have CI (since it runs `yarn test`, which in turn runs `yarn lint`) auto-fix potential errors and not commit them anywhere.

Reply:
good call.